sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/OpenAIAsyncClient.java

```diff
@@ -417,9 +417,9 @@ public Mono<Completions> getCompletions(String deploymentId, CompletionsOptions
      *     that continues from or "completes" provided prompt data on successful completion of {@link Mono}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Completions> getCompletions(String deploymentId, List<String> prompts) {
+    public Mono<Completions> getCompletions(String deploymentId, String prompt) {
```

Comment:
We don't need to add the `getCompletionsWithResponse`  method above that returns the BinaryData for a single prompt.